src/metatensor/models/experimental/alchemical_model/train.py

```diff
@@ -33,20 +35,36 @@
 
 
 def train(
-    train_datasets: List[Union[_BaseDataset, torch.utils.data.Subset]],
-    validation_datasets: List[Union[_BaseDataset, torch.utils.data.Subset]],
-    requested_capabilities: ModelCapabilities,
+    train_datasets: List[Union[Dataset, torch.utils.data.Subset]],
+    validation_datasets: List[Union[Dataset, torch.utils.data.Subset]],
+    dataset_info: DatasetInfo,
     hypers: Dict = DEFAULT_HYPERS,
     continue_from: Optional[str] = None,
     output_dir: str = ".",
     device_str: str = "cpu",
 ):
+    all_species = get_all_species(train_datasets + validation_datasets)
+    outputs = {
+        key: ModelOutput(
+            quantity=value.quantity,
+            unit=value.unit,
+            per_atom=value.per_atom,
```

Comment:
Just to check, is alchemical model able to do per-atom predictions? Or not? If it is, this should be `per_atom=True`.

And a separate question: is the model able to train with non-energy targets & per-atom targets?

Reply:
You're right, per-atom should be false, here and in the SOAP-BPNN. For now, we only support energy targets for all models.
Related: in order to support other targets, we first need to support their quantities in `metatensor.torch.atomistic`, right?

Reply:
>  Related: in order to support other targets, we first need to support their quantities in metatensor.torch.atomistic, right?

yes!